client/implementation/src/main/java/io/smallrye/graphql/client/impl/typesafe/json/JsonReader.java

```diff
@@ -45,10 +45,6 @@ Object read() {
         if (isListOfErrors(value) && !isGraphQlErrorsType())
             throw cantApplyErrors(readGraphQlClientErrors());
         Reader<?> reader = reader(location);
-        if (reader == null) {
-            throw new InvalidResponseException(
-                    "invalid " + type.getTypeName() + " value for " + location.getDescription() + ": " + value);
-        }
```

Comment:
@jmartisk: these changes are unrelated to this ticket but we discussed about this in another ticket.